compiler/ras/DebugCounter.hpp

```diff
@@ -139,7 +140,8 @@ class DebugCounter : public DebugCounterBase
    //
    const char      *getName()                  { return _name; }
    int8_t           getFidelity()              { return _fidelity; }
-   int64_t          getCount()                 { return _totalCount; }
+   /** On 32-bit platforms, the lower word of _totalCount will be zero, since we use 32-bit operations*/
```

Comment:
Is this an endian specific comment? if so it should be made endian agnostic since the behaviour here will work regardless of endianness - perhaps just at the opposite end of the memory